public/scripts/helper.js

```diff
@@ -0,0 +1,25 @@
+// https://stackoverflow.com/questions/14573223/set-cookie-and-get-cookie-with-javascript
+export function setCookie(name,value,days) {
+    var expires = "";
+    if (days) {
+        var date = new Date();
```

Comment:
I would prefer the usage of let as often as possible over var, as var's are outdated
```suggestion
        let date = new Date();
```